docs/user-guide/helm.md

```diff
@@ -164,6 +164,17 @@ Or via declarative syntax:
           value: $ARGOCD_APP_NAME
 ```
 
+It's also possible to use build environment variables for the Helm values file path:
+
+```yaml
+  spec:
+    source:
+      helm:
+        valueFiles:
+              - values.yaml
+              - myprotocol://somepath/$ARGOCD_APP_NAME/$ARGOCD_APP_REVISION
```

Comment:
Tiiiiny nitpick. Could the indentation be shifted left? I think this is valid yaml, just looks a bit odd.  :-)

Reply:
Good point! aligned it now with the parameters example above.